v2/googlecloud-to-elasticsearch/docs/GCSToElasticsearch/README.md

```diff
@@ -11,11 +12,35 @@ If either a UDF or JSON schema is provided then it will be used instead of the C
 
 \*<b>NOTE:</b> All values will be written to Elasticsearch as strings if headers are used.
 
-Pipeline flow is illustrated below:
+## Getting Started
 
-![alt text](../img/gcs-to-elasticsearch-dataflow.png "GCS to Elasticsearch pipeline flow")
+The simplest way to set up Elasticsearch is to create a managed deployment with Elasticsearch Service on [Elastic Cloud](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started.html#run-elasticsearch). You can deploy Elasticsearch on many [Google Cloud regions](https://www.elastic.co/guide/en/cloud/current/ec-regions-templates-instances.html) globally. If you prefer to manage your own environment, you can install and run [Elasticsearch using Docker](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started.html#run-elasticsearch).
 
-## Getting Started
+### Parameters
+
+The template requires the following parameters:
+* connectionUrl: Elasticsearch URL in format http://hostname:[port] or CloudId
+* inputFileSpec: Pattern to where data lives in GCS. Example: `gs://mybucket/somepath/*.csv`
+* errorOutputTopic: Error output topic in Pub/Sub for failed inserts
```

Comment:
In the current template errors are routed to BQ.  errorOutputTopic is not relevant and should be removed